cardano-node/cardano-node.cabal

```diff
@@ -140,7 +140,7 @@ executable cardano-node
   ghc-options:         -threaded
                        -Wall
                        -rtsopts
-                       "-with-rtsopts=-T -I0"
+                       "-with-rtsopts=-T -I0 -N2 -A16m"
```

Comment:
I am not sure this will help. The node is run in various scripts with different RTS flags.
Do we need this here?

Reply:
Yes, it needs to be run with those flags.
But if it is bypassed in scripts somewhere it doesn't do much good.

Reply:
In my experiments, having `-N2` (or greater) has been very important for performance.